griffon/commands/entities/osidb/affects/cvss.py

```diff
@@ -0,0 +1,229 @@
+"""
+osidb affect CVSS entities operations
+"""
+
+import json
+import logging
+
+import click
+from osidb_bindings.bindings.python_client.api.osidb import (
+    osidb_api_v1_affects_cvss_scores_create,
+    osidb_api_v1_affects_cvss_scores_list,
+    osidb_api_v1_affects_cvss_scores_retrieve,
+    osidb_api_v1_affects_cvss_scores_update,
+)
+from osidb_bindings.bindings.python_client.models import AffectCVSS
+from requests import HTTPError
+
+from griffon import OSIDB_API_URL, OSIDBService, progress_bar
+from griffon.commands.entities.helpers import (
+    abort_if_false,
+    filter_request_fields,
+    get_editor,
+    multivalue_params_to_csv,
+    query_params_options,
+    request_body_options,
+)
+from griffon.output import console, cprint
+
+logger = logging.getLogger("griffon")
+
+
+@click.group(help=f"{OSIDB_API_URL}/osidb/api/v1/affects/<id>/cvss_scores", name="cvss")
+@click.pass_context
+def affect_cvss(ctx):
+    """OSIDB Affect CVSS."""
+    pass
+
+
+@affect_cvss.command(name="get")
+@click.option(
+    "--affect-id",
+    "affect_id",
+    help="affect CVE-ID or UUID.",
+    required=True,
+)
+@click.option("--uuid", "cvss_uuid", help="CVSS UUID.", required=True)
+@query_params_options(
+    entity="affect CVSS",
+    endpoint_module=osidb_api_v1_affects_cvss_scores_retrieve,
+    options_overrides={
+        "include_fields": {"type": click.Choice(OSIDBService.get_fields(AffectCVSS))},
+        "exclude_fields": {"type": click.Choice(OSIDBService.get_fields(AffectCVSS))},
+        "include_meta_attr": {"type": click.Choice(OSIDBService.get_meta_attr_fields(AffectCVSS))},
+    },
+)
+@click.pass_context
+@progress_bar
+def get_affect_cvss(ctx, affect_id, cvss_uuid, **params):
+    params = multivalue_params_to_csv(params)
+
+    session = OSIDBService.create_session()
+    data = session.affects.cvss_scores.retrieve(affect_id, cvss_uuid, **params)
+    return cprint(data, ctx=ctx)
+
+
+@affect_cvss.command(name="list")
+@click.option(
+    "--affect-id",
+    "affect_id",
+    help="affect CVE-ID or UUID.",
+    required=True,
+)
+@query_params_options(
+    entity="affect CVSS",
+    endpoint_module=osidb_api_v1_affects_cvss_scores_list,
+    options_overrides={
+        "include_fields": {"type": click.Choice(OSIDBService.get_fields(AffectCVSS))},
+        "exclude_fields": {"type": click.Choice(OSIDBService.get_fields(AffectCVSS))},
+        "include_meta_attr": {"type": click.Choice(OSIDBService.get_meta_attr_fields(AffectCVSS))},
+    },
+)
+@click.pass_context
+@progress_bar
+def list_affect_cvss(ctx, affect_id, **params):
+    # TODO: handle pagination
```

Comment:
minor - I believe we handle pagination and output today

Reply:
I don't think we handle dynamic output (eg. when due to a include/exclude fields some fields are omitted. We have a issue for that.

Reply:
no but we do handle output and pagination ... anyhow only nitpicking